src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs

```diff
@@ -209,6 +211,58 @@ public async Task Get_MemberWithNoActivity_ShouldReturnExpectedValueForActivitie
         });
     }
 
+    [Test, AutoData]
+    public async Task Get_NewApprenticeAccount_ShouldReturnExpectedValueForReasonForJoining(MemberProfileResponse memberProfileResponse, GetProfilesResult profilesResult, string reasonToJoin, Mock<ISessionService> _sessionServiceMock, Mock<IOuterApiClient> _outerApiClientMock, CancellationToken _cancellationToken)
```

Comment:
Coverage for scenario where Apprentice user who is not new should be added as well.

Reply:
Done